### PR TITLE
Get portfolios tdd

### PR DESCRIPTION
**Pull request by ndara**
Implemented GET /portfolios

### CODE REVIEW

**Comment by asnewman** [comment]
Is there a reason you always name these PRs 'something tdd' instead of the jira task name?

File: Service/src/routes/portfolio.py [hunk 1]
```diff
@@ -30,6 +30,46 @@ def post_portfolio():
...
+   items = []
+   g.cursor.execute("SELECT id, shareCount, avgCost, ticker FROM PortfolioItem WHERE portfolioId = %s", portfolio['id'])
+   items = g.cursor.fetchall()
+
+   portfolio['items'] = items
+
+
+def attach_portfolio_netWorth(portfolio):
+   netWorth = float(portfolio['buyPower'])
+   if len(portfolio['items']) > 0:
```
Comment (asnewman) [inline Service/src/routes/portfolio.py]: Is this check necessary? Wouldn't it just skip the for loop if the length isn't greater than 0?

**Review by asnewman — CHANGES_REQUESTED** [review]
Just the if statement question

File: Service/src/routes/portfolio.py (hunk 1 above)
Comment (ndara) [reply Service/src/routes/portfolio.py]: yea I guess that's true huh

**Review by asnewman — APPROVED** [review]

**Pull request merged**